### ISSUE TITLE
Inverse of a quantity

### ISSUE THREAD

**Issue by bharathdgowda** [angularsen/UnitsNet#931]
There doesn’t seem to be a straightforward way to get the inverse of a quantity (where applicable).

For example, I’d like to easily compute the inverse of a Conductivity quantity to get the Resistivity, while being confident that the conversion factor & units are consistent. I guess there are other such quantities that have such an inverse relationship as well.

I’ve tried converting the quantity to SI units, calculating 1/n of the value and creating the inverse quantity. But I’m struggling to convert it back to the equivalent unit I started with.

Having this functionality built-in to the quantity types would be great!

**Comment by angularsen** [comment]
Hi, I'm not sure if this can be resolved in a generic way.
The strength of UnitsNet is static typing, so any inversion calculations would have to be explicitly implemented.

Custom conversion methods between quantities are implemented as operator overload methods, like this one:

```c#
public static Length operator *(Duration duration, Speed speed)        
{            
    return Length.FromMeters(speed.MetersPerSecond * duration.Seconds);        
}
```

https://github.com/angularsen/UnitsNet/blob/master/UnitsNet/CustomCode/Quantities/Speed.extra.cs.

So, if I understand your question correctly, a similar inversion method could be implemented between https://github.com/angularsen/UnitsNet/blob/master/Common/UnitDefinitions/ElectricResistivity.json and https://github.com/angularsen/UnitsNet/blob/master/Common/UnitDefinitions/ElectricConductivity.json.

Did that answer your question?

**Comment by angularsen** [comment]
Fixed by #943
